zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedOrErrorRuntimeChecker.scala

```diff
@@ -0,0 +1,71 @@
+package com.broadinstitute.dsp
+package zombieMonitor
+
+import cats.effect.{Concurrent, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import com.broadinstitute.dsp.CloudService.{Dataproc, Gce}
+import fs2.Stream
+import io.chrisdavenport.log4cats.Logger
+import org.broadinstitute.dsde.workbench.google2.{DataprocClusterName, InstanceName}
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+/**
+ * Take a `Stream` of active runtimes, and check their status in google
+ * 1. if non-existent in google, update DB to "Deleted"
+ * 2. if exists but in Error status in google, update DB to "Error"
+ * 3. else, do nothing
+ */
+object DeletedOrErrorRuntimeChecker {
```

Comment:
```suggestion
object DeletedOrErroredRuntimeChecker {
```
similarly to `DeletedOrErroredNodepoolChecker`